package.json

```diff
@@ -23,6 +23,7 @@
     "@theforeman/vendor": "^4.0.7",
     "intl": "~1.2.5",
     "jed": "^1.1.1",
+    "react-helmet": "^5.2.1",
```

Comment:
https://github.com/theforeman/foreman-js/pull/124

Reply:
foreman-js pr is merged

Reply:
updated